test/unit/lifecycle/attribute-changed.js

```diff
@@ -18,4 +19,24 @@ describe('lifecycle/attribute-changed', function () {
       }
     });
   });
+
+  it('attributes that are defined as properties should call attributeChanged callback', function (done) {
+    let counter = 0;
+    let elem = new (element().skate({
+      attributeChanged () {
+        counter++;
+      },
+      props: {
+        test: {
+          attribute: true
+        },
+      },
+    }));
+    afterMutations(done);
```

Comment:
You might want to make assertions within `afterMutations` and calling `done` last, like:

```js
afterMutations(
  () => expect...,
  () => elem.test = ...,
  ...
  done
);
```